lm_eval/tasks/med_concepts_qa/_med_concepts_qa.yaml

```diff
@@ -0,0 +1,7 @@
+group: med_concepts_qa
+task:
+  - icd9cm
+  - icd10cm
```

Comment:
I think these task names will be confusing to outsiders. Could we prefix them all with `med_concepts_qa_...` ?